src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1585,4 +1557,4 @@
     <value>Set as default</value>
     <comment>Text label for a button that, when invoked, sets the selected color scheme as the default scheme to use.</comment>
   </data>
-</root>
+</root>
```

Comment:
If you saved this with a text editor, you may need to go re-save it with Visual Studio ;)

Reply:
hah! Already did via VS! I know better than that :P